setup.py

```diff
@@ -45,7 +41,7 @@
       package_data={'gridData': ['tests/datafiles/*.dx', 'tests/datafiles/*.dx.gz',
                                  'tests/datafiles/*.ccp4', 'tests/datafiles/*.plt',
                                  'tests/datafiles/*.bz2']},
-      install_requires=['numpy>=1.0.3', 'six', 'scipy', 'mrcfile'],
+      install_requires=['numpy>=1.19', 'scipy', 'mrcfile'],
```

Comment:
I bumped numpy because of NEP29 but it’s not really necessary. Could be reverted to better show what’s minimally needed.

Reply:
My suggestion would be to follow NEP29 as closely as possible, if there ends up being issues further down the line then it can be relaxed?

Reply:
Ok, thanksgiving